src/app/app-routing.module.ts

```diff
@@ -1,16 +1,18 @@
-import { NgModule } from "@angular/core";
-import { Routes, RouterModule } from "@angular/router";
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
 
 const routes: Routes = [
 	{
-		path: "",
-		loadChildren: () => import("./starter-home/starter-home.module").then(m => m.StarterHomeModule)
+		path: '',
+		loadChildren: () => import('./home/home.module').then(m => m.HomeModule),
 	},
 	{
-		path: "",
-		redirectTo: "",
-		pathMatch: "full"
-	}
+		path: 'repos',
+		loadChildren: () =>
+			import('./repositories/repositories.module').then(
```

Comment:
Indentation looks weird here.

Reply:
Should I not indent?

Reply:
It could be one line, but not what I meant :)